index.html

```diff
@@ -49,6 +50,182 @@ <h2>LET’S CONNECT</h2>
 
+    <section class="work">
+      <div class="card">
+        <img src="assets/images/mobile/portfolio1.png" alt="project1">
+        <div class="card-text">
+          <div class="card-header">
+            <h3>Tonic</h3>
+            <div class="frame">
+              <ul>
+                <li class="client"><h4>CANOPY</h4></li>
+                <li class="role"><i class="fa fa-circle"></i><h4>Back End Dev</h4></li>
+                <li class="year"><i class="fa fa-circle"></i><h4>2015</h4></li>
+              </ul>
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="tag">
+                <ul>
+                  <li class="tag1"><h3>HTML</h3></li>
+                  <li class="tag2"><h3>CSS</h3></li>
+                  <li class="tag3"><h3>Javascript</h3></li>
+                </ul>
+              </div>
+              <div class="see-more-detail">
+                <div class="button">
+                  <a href="#">
+                    See Project
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="card">
+        <img src="assets/images/mobile/portfolio3.svg" alt="project1">
+        <div class="card-text">
+          <div class="card-header">
+            <h3>Tonic</h3>
+            <div class="frame">
+              <ul>
+                <li class="client"><h4>CANOPY</h4></li>
+                <li class="role"><i class="fa fa-circle"></i><h4>Back End Dev</h4></li>
+                <li class="year"><i class="fa fa-circle"></i><h4>2015</h4></li>
+              </ul>
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="tag">
+                <ul>
+                  <li class="tag1"><h3>HTML</h3></li>
+                  <li class="tag2"><h3>CSS</h3></li>
+                  <li class="tag3"><h3>Javascript</h3></li>
+                </ul>
+              </div>
+              <div class="see-more-detail">
+                <div class="button">
+                  <a href="#">
+                    See Project
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="card2">
+        <img src="assets/images/mobile/portfolio4.svg" alt="image">
+        <div class="card-text2">
+          <div class="card-header2">
+            <h3>Multi-Post
+              Stories</h3>
+            <div class="frame2">
+              <ul>
+                <li class="client"><h4>CANOPY</h4></li>
+                <li class="role"><i class="fa fa-circle"></i><h4>Back End Dev</h4></li>
+                <li class="year"><i class="fa fa-circle"></i><h4>2015</h4></li>
+              </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- [ ] Your font for the last two paragraphs of the projects look different from others and the template. Please, use the correct font.

![emma-font](https://user-images.githubusercontent.com/36542986/134599229-4f6d5f61-e3cc-4f41-b941-590517c9d548.jpg)